src/main/java/esubine/community/comment/CommentService.java

```diff
@@ -56,12 +56,14 @@ public EmptyResponse updateComment(Long userId, Long commentId, CommentRequest c
 //        return comment;
     }
 
+
     public EmptyResponse deleteComment(Long userId, Long commentId) {
-        Optional<CommentEntity> commentEntityOptional = commentRepository.findById(commentId);
-        CommentEntity comment = commentEntityOptional.orElseThrow(() -> new NoDataException("댓글을 찾을 수 없습니다."));
+        Optional<CommentEntity> commentOptional = commentRepository.getByCommentId(commentId);
+        CommentEntity comment = commentOptional.orElseThrow(() -> new NoDataException("댓글을 찾을 수 없습니다."));
```

Comment:
코드 한 줄로 합치면 좋을 것 같습니다.
```suggestion
        CommentEntity comment  = commentRepository.getByCommentId(commentId).orElseThrow(() -> new NoDataException("댓글을 찾을 수 없습니다."));
```